src/game/common/rts/science.h

```diff
@@ -23,7 +23,7 @@
 #include "unicodestring.h"
 #include <vector>
 
-enum ScienceType : int32_t
+enum ScienceType
```

Comment:
Other game enums use default type, so might as well make it consistent here.

Reply:
afaik you need to specify size if you forward declare them. That was why I added the size.

Reply:
Forward declare without explicit type should be fine (unless there is some weird compiler?). Enums have a default type (int) and will use that when not specified.